src/ROUTES.ts

```diff
@@ -109,6 +109,7 @@ const ROUTES = {
     },
     SETTINGS_ADD_DEBIT_CARD: 'settings/wallet/add-debit-card',
     SETTINGS_ADD_BANK_ACCOUNT: 'settings/wallet/add-bank-account',
+    SETTINGS_ADD_BANK_ACCOUNT_REFACTOR: 'settings/wallet/add-bank-account-refactor',
```

Comment:
The name 'refactor' feels odd. Will SETTINGS_ADD_BANK_ACCOUNT continue to exist? Maybe we should consider incorporating versioning into the naming, like `add-bank-account-v2`.

Reply:
I will remove the REFACTOR route and connect with the new flow when everything is done. It is now added for testing and development purpose :)